sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs

```diff
@@ -9,40 +9,52 @@
 
 namespace Azure.Containers.ContainerRegistry
 {
-    /// <summary> The registry service client. </summary>
+    /// <summary> The Azure Container Registry service client. </summary>
     public partial class ContainerRegistryClient
     {
-        private readonly Uri _endpoint;
+        private readonly Uri _registryUri;
+        private readonly string _registryName;
         private readonly HttpPipeline _pipeline;
         private readonly HttpPipeline _acrAuthPipeline;
         private readonly ClientDiagnostics _clientDiagnostics;
         private readonly ContainerRegistryRestClient _restClient;
-
         private readonly AuthenticationRestClient _acrAuthClient;
         private readonly string AcrAadScope = "https://management.core.windows.net/.default";
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="ContainerRegistryClient"/> for managing container images and artifacts.
         /// </summary>
-        public ContainerRegistryClient(Uri endpoint, TokenCredential credential) : this(endpoint, credential, new ContainerRegistryClientOptions())
+        /// <param name="registryUri">The URI endpoint of the container registry.  This is likely to be similar
+        /// to "https://{registry-name}.azurecr.io".</param>
+        /// <param name="credential">The API key credential used to authenticate requests
+        /// against the container registry.  </param>
+        public ContainerRegistryClient(Uri registryUri, TokenCredential credential) : this(registryUri, credential, new ContainerRegistryClientOptions())
         {
         }
 
         /// <summary>
+        /// Initializes a new instance of the ContainerRegistryClient for managing container images and artifacts.
         /// </summary>
-        public ContainerRegistryClient(Uri endpoint, TokenCredential credential, ContainerRegistryClientOptions options)
+        /// <param name="registryUri">The URI endpoint of the container registry.  This is likely to be similar
+        /// to "https://{registry-name}.azurecr.io".</param>
+        /// <param name="credential">The API key credential used to authenticate requests
+        /// against the container registry.  </param>
+        /// <param name="options">Client configuration options for connecting to Azure Container Registry.</param>
+        public ContainerRegistryClient(Uri registryUri, TokenCredential credential, ContainerRegistryClientOptions options)
```

Comment:
Document exceptions - in the other constructor as well.

Reply:
Will do!